lib/grape.rb

```diff
@@ -27,6 +27,17 @@
 require 'i18n'
 require 'thread'
 
+CONTENT_TYPE = 'Content-Type'.freeze
```

Comment:
I don't like how these are globals. They are all HTTP headers, so create a module, like `Grape::HTTP::Headers` and stick them in there?